soc/Kconfig

```diff
@@ -0,0 +1,9 @@
+
+choice
+	prompt "SoC/CPU/Configuration Selection"
+	source "soc/$ENV_VAR_ARCH/*/Kconfig.soc"
```

Comment:
Small subjective nit here. Could make it:

```
choice
	prompt "SoC/CPU/Configuration Selection"

source "soc/$ENV_VAR_ARCH/*/Kconfig.soc"

endchoice
```

Avoids confusing the `source` with a property on the `choice`.

Reply:
(Unrelated: I wonder why it's `ENV_VAR_ARCH` instead of just `ARCH`.)